### PR TITLE
Installer instructions

### PR DESCRIPTION
**Pull request by willihay**
Signed-off-by: William Hayward <wilhayw@amazon.com>

### CODE REVIEW

File: content/docs/welcome-guide/setup/setup-from-installer.md [hunk 1]
```diff
@@ -2,8 +2,50 @@
...
+* Met all hardware and software requirements listed in [System Requirements](./requirements).
+* Configured the required software as described in [Software Configuration](./requirements#software-configuration).
+
+## Installing O3DE
+
+1. Obtain the latest version of the installer from the [O3DE download](https://o3de.org/download/) page.
+
+1. Run the installer from your download location.
+
+1. During developer preview, you might encounter a warning such as the following from Microsoft Defender describing the program as an unrecognized app. To proceed with installation, you must first open the **More info** link and then choose **Run anyway**.
```
Comment (paucom) [inline content/docs/welcome-guide/setup/setup-from-installer.md:25]:
```suggestion
1. During the developer preview, you might encounter the following warning from Microsoft Defender describing the program as an unrecognized app. To proceed with installation, choose **More info**, and then choose **Run anyway**.
```
The current wording describes a warning "such as" this Microsoft Defender sideloading warning. Might the reader receive a similar warning? If not, see my suggested changes above. If so, I recommend something like the following wording instead for clarity:
>During the developer preview, you might encounter a warning about sideloading the app, such as the following warning from Microsoft Defender. If you get this particular warning, to proceed with installation, choose **More info**, and then choose **Run anyway**.

File: content/docs/welcome-guide/setup/setup-from-installer.md [hunk 2]
```diff
@@ -2,8 +2,50 @@
...
+
+    ![Microsoft Defender Windows protection dialog boxes](/images/welcome-guide/installer-defender-protection.png)
+
+1. The default location for the engine is: `C:\O3DE\<o3de_version>`. Choose **Install** to install O3DE here, or choose **Options** to change the install location.
```
Comment (paucom) [inline content/docs/welcome-guide/setup/setup-from-installer.md:29]:
```suggestion
1. The default install location is: `C:\O3DE\<o3de_version>`. To install O3DE here, choose **Install**. To change the install location, choose **Options**.
```

File: content/docs/welcome-guide/setup/setup-from-installer.md [hunk 3]
```diff
@@ -2,8 +2,50 @@
...
+
+    ![O3DE welcome and options](/images/welcome-guide/installer-welcome.png)
+
+1. During installation, additional packages will be downloaded as needed. The process can take some time, depending on your internet connection speed. You might notice additional windows open and close during install - for example, when the Python runtime is installed in the engine's `python` directory.
```
Comment (paucom) [inline content/docs/welcome-guide/setup/setup-from-installer.md:33]:
```suggestion
1. During installation, the installer downloads additional packages as needed. The process can take some time, depending on your internet connection speed. Note that additional windows might open and close during installation—for example, when installing the Python runtime in the engine's `python` directory.
```
* Some passive/future voice edits (to active/present)
* Minor edits for flow, punctuation conventions

File: content/docs/welcome-guide/setup/setup-from-installer.md [hunk 4]
```diff
@@ -2,8 +2,50 @@
...
+
+    ![O3DE install progress](/images/welcome-guide/installer-install-progress.png)
+
+1. The installer displays an **Installation Successfully Completed** message if installation was successful. Choose the **Launch** button to open Project Manager, or **Close** to exit the installer.
```
Comment (paucom) [inline content/docs/welcome-guide/setup/setup-from-installer.md:37]:
```suggestion
1. Upon successful installation, the installer displays the message **Installation Successfully Completed**. To open Project Manager, choose **Launch**. To exit the installer, choose **Close**.
For improved comprehension, rewrote the sentences to lead with the intent/infinitive phrases.
```

File: content/docs/welcome-guide/setup/setup-from-installer.md [hunk 5]
```diff
@@ -2,8 +2,50 @@
...
+
+    ![O3DE install successful](/images/welcome-guide/installer-completed-success.png)
+
+1. The installer creates two shortcuts on the desktop for common O3DE applications:
```
Comment (paucom) [inline content/docs/welcome-guide/setup/setup-from-installer.md:41]:
```suggestion
The installer creates two shortcuts on the desktop for common O3DE applications:
```
This isn't really a step of installation, but rather a description of some results of the prior steps. Consider dropping the numbering and just making this a new, regular paragraph.

File: content/docs/welcome-guide/setup/setup-from-installer.md [hunk 6]
```diff
@@ -2,8 +2,50 @@
...
+
+    ![O3DE Editor icon](/images/welcome-guide/desktop-icon-editor.png) **O3DE Editor**
+
+    The Editor is a central hub for O3DE's creative tools. Place and group entities, add components, configure properties, and open supporting tools to help you implement your project goals. Take the [Editor Tour](/docs/welcome-guide/get-started/editor-tour) to learn more about this application.
```
Comment (paucom) [inline content/docs/welcome-guide/setup/setup-from-installer.md:45]:
```suggestion
    Editor is a central hub for O3DE's creative tools. Place and group entities, add components, configure properties, and open supporting tools to help you implement your project goals. To learn more about Editor, take the [Editor tour](/docs/welcome-guide/get-started/editor-tour).
```
* It's unclear if the phrase "to help you implement your project goals" describes the whole series of Editor features in that sentence, or just the "supporting tools". If the former, move this intent/infinitive phrase to the front of the sentence ("To help you implement..., place and group entities, ....") If the latter, change "to" to "that" ("supporting tools that help you....").
* Per our [terminology guidelines](https://docs.o3de.org/docs/contributing/to-docs/terminology/), use just "Editor" as a proper noun. (Drop the "the".)
* Made "tour" lower case, as that's not actually the title of the linked "tour" page. (Alternatively, we can retitle that page to include "Tour" in the title.)

File: content/docs/welcome-guide/setup/setup-from-installer.md [hunk 7]
```diff
@@ -2,8 +2,50 @@
...
+
+    ![O3DE Project Manager icon](/images/welcome-guide/desktop-icon-project-manager.png) **O3DE Project Manager**
+
+    Create and customize projects using the Project Manager tool. You can also enable or disable Gems to add or remove functionality for your project.
```
Comment (paucom) [inline content/docs/welcome-guide/setup/setup-from-installer.md:49]:
```suggestion
    Create and customize projects using the Project Manager tool. To add or remove functionality for your project, you can also enable or disable Gems.
```

File: content/docs/welcome-guide/setup/setup-from-installer.md [hunk 8]
```diff
@@ -2,8 +2,50 @@
...
+
+    For an introduction to Project Manager and help creating your first project, continue to [Creating Projects Using the Open 3D Engine Project Manager](/docs/welcome-guide/get-started/project-config/creating-projects-using-project-manager) in this guide.
```
Comment (paucom) [inline content/docs/welcome-guide/setup/setup-from-installer.md:51]:
```suggestion
    For an introduction to Project Manager and help creating your first project, refer to [Creating Projects Using the Open 3D Engine Project Manager](/docs/welcome-guide/get-started/project-config/creating-projects-using-project-manager).
```

**Review by paucom — COMMENTED** [review]
Copyedits

**Review by sptramer — APPROVED** [review]

File: content/docs/welcome-guide/setup/requirements.md [hunk 9]
```diff
@@ -6,7 +6,7 @@ weight: 100
 toc: true
 ---
 
-Open 3D Engine (O3DE) has a minimum set of hardware and software requirements for development, as outlined in the following sections.
+Open 3D Engine (O3DE) has a minimum set of hardware and software requirements for development, as outlined in the following sections. In addition, some of the software requires specific configurations, as detailed in the last section.
```
[the remark this answers is not in the document]
Comment (willihay) [reply content/docs/welcome-guide/setup/requirements.md:9]: Edited your suggestion to correct the link.

File: content/docs/welcome-guide/setup/setup-from-installer.md [hunk 10]
```diff
@@ -2,8 +2,50 @@
...
 description: Learn how to install and set up Open 3D Engine (O3DE) using the installer tool.
 weight: 200
+draft: true
 ---
 
-{{< placeholder >}}
+A quick way to get started with O3DE is to download and install a released version. After running the installer, you will have a stable, pre-built version of the engine and its Gems, ready for you to jump into your next project.
 
-Coming soon! For now, follow the instructions for installing O3DE from GitHub found in [Setup O3DE from GitHub](./setup-from-github).
+The instructions here will guide you through the installation experience. At the end of a successful install, you will be ready to create new or open existing projects with the **Project Manager** tool.
```
[the remark this answers is not in the document]
Comment (willihay) [reply content/docs/welcome-guide/setup/setup-from-installer.md:10]: Ooh, good wordsmithing! Slick.

File: content/docs/welcome-guide/setup/setup-from-installer.md (hunk 1 above)
Comment (willihay) [reply content/docs/welcome-guide/setup/setup-from-installer.md:25]: When I wrote this, I wasn't sure if we wanted to allude to scenarios where a reader is using something besides Defender, and therefore might encounter a different warning. In retrospect, I think this unnecessarily complicates this text, which is only temporary during the developer preview anyway. I'll go with your first suggestion.

File: content/docs/welcome-guide/setup/setup-from-installer.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (willihay) [reply content/docs/welcome-guide/setup/setup-from-installer.md:10]: Not sure why I can't commit this suggestion. I'll change in a separate commit.

File: content/docs/welcome-guide/setup/setup-from-installer.md [hunk 11]
```diff
@@ -2,8 +2,48 @@
...
+
+    ![O3DE Editor icon](/images/welcome-guide/desktop-icon-editor.png) **O3DE Editor**
+
+    Editor is a central hub for O3DE's creative tools. To help you implement your project goals, use Editor to place and group entities, add components, configure properties, and open supporting tools such as **Animation Editor** and **Script Canvas**. To learn more about Editor, take the [Editor tour](/docs/welcome-guide/get-started/editor-tour).
+
+    ![O3DE Project Manager icon](/images/welcome-guide/desktop-icon-project-manager.png) **O3DE Project Manager**
+
+    Create and customize projects using the Project Manager tool. To add or remove functionality for your project, you can also enable or disable Gems.
+
+    For an introduction to Project Manager and help creating your first project, continue to [Creating Projects Using the Open 3D Engine Project Manager](/docs/welcome-guide/get-started/project-config/creating-projects-using-project-manager) to take an introductory tutorial.
```
Comment (willihay) [inline content/docs/welcome-guide/setup/setup-from-installer.md:49]: @paucom I like the use of **continue** in this sentence to give a sense of continuity and direction.

Also, I know you like to simplify with infinitives. In this case, to achieve that feeling of direction, I like the structure of "For [this], do [this]." What do you think?

**Review by paucom — APPROVED** [review]
LG!

**Pull request merged**